docs/user_guide.md

```diff
@@ -69,8 +71,13 @@ output.
 ## Default configuration
 
 Used ports:
+
 * `5432` - PostgreSQL
-* `8001` - CodeChecker result viewer
+* `8001` - CodeChecker server
+
+The server listens only on the local machine.
```

Comment:
That is why it says **Default** configuration. If someone checks the help for `server`, they will see.